src/www/interfaces.php

```diff
@@ -598,7 +599,7 @@ function get_wireless_channel_info($interface) {
         if (isset($config['dhcpd']) && isset($config['dhcpd'][$if]['enable']) && (! preg_match("/^staticv4/", $pconfig['type']))) {
             $input_errors[] = gettext("The DHCP Server is active on this interface and it can be used only with a static IP configuration. Please disable the DHCP Server service on this interface first, then change the interface configuration.");
         }
-        if (isset($config['dhcpdv6']) && isset($config['dhcpdv6'][$if]['enable']) && (! preg_match("/^staticv6/", $pconfig['type6']))) {
+        if (isset($config['dhcpdv6']) && isset($config['dhcpdv6'][$if]['enable']) && (! preg_match("/^staticv6/", $pconfig['type6'])) && !isset($config['system']['dhcpd6track6allowoverride'])) {
```

Comment:
Note to self: this validation is very strange. we should opt for a clean service change, not block a reconfigure. Also, this might have been fixed via a0bb26b529b

Reply:
What's even stranger is this one has sneaked through with $config['system'], so much for a global file search!